### PR TITLE
Final Products page.

### PR DESCRIPTION
**Pull request by Modern-artist**
**Description**
Final Products page.

This PR fixes #
Implemented Final Product page.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 112e4b19cfa89144e72283d461fb984bf8f846c0 at: https://6543202a8fec0a04d411d398--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
Hey @Modern-artist  Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by l5io** [comment]
🚀 Preview for commit 18c6c418132400f96a8d801b3411d5eba859786a at: https://6548c4b886d5720cb35882e8--layer5.netlify.app

**Comment by saurabh100ni** [comment]
Please add this as your agenda in [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) to present it in the Website Meet today at 6:30 pm IST/7:00 am CT

**Review by ankushbhardwaj408 — COMMENTED** [review]
When we hover over product link in navbar the whole screen turns white

**Comment by abhijeetgauravm** [comment]
> When we hover over product link in navbar the whole screen turns white

Yeah @ankushbhardwaj408  // @Modern-artist

**Comment by l5io** [comment]
🚀 Preview for commit a1173fe68e132e00d9f4379ef909253f11755d4c at: https://654bacd913188767397f3dbc--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
> When we hover over product link in navbar the whole screen turns white

@Modern-artist

**Comment by leecalcote** [comment]
@Modern-artist, it means that you have a bug.

File: src/assets/images/products/MesheryRange.svg
Comment (leecalcote) [inline src/assets/images/products/MesheryRange.svg:1]: These images should be colocated in the same components in which they are used.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/assets/images/products/MesheryRangeDark.svg
Comment (leecalcote) [inline src/assets/images/products/MesheryRangeDark.svg:1]: This SVG contains a number of PNGs and is not usable. Ensure only vector contents is used.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/assets/images/products/Nightawk.svg
Comment (leecalcote) [inline src/assets/images/products/Nightawk.svg:1]: This image and other images here are really just PNGs and cannot be used.

File: src/assets/images/products/circle_dark.svg
Comment (leecalcote) [inline src/assets/images/products/circle_dark.svg:1]: Relocate this image.

File: src/assets/images/products/patterns.svg
Comment (leecalcote) [inline src/assets/images/products/patterns.svg:1]: We already have this logo in this site's general assets. Reuse.

File: src/assets/images/products/meshmap.svg
Comment (leecalcote) [inline src/assets/images/products/meshmap.svg:1]: This is a PNG.

Comment (leecalcote) [reply src/assets/images/products/meshmap.svg:1]: We already have this logo in this site's general assets. Reuse.

File: src/pages/products/index.js
Comment (leecalcote) [inline src/pages/products/index.js:1]: Like other pages, this page should be in sections and contain both sections and components.

File: src/pages/products/icons/support.svg
Comment (leecalcote) [inline src/pages/products/icons/support.svg:1]: Use the existing Support icon from Cloud, which is Meshery Operator.

File: src/pages/products/icons/meshmap-icon.svg
Comment (leecalcote) [inline src/pages/products/icons/meshmap-icon.svg:1]: Don't duplicate.

File: src/pages/products/icons/coming-soon.webp
Comment (leecalcote) [inline src/pages/products/icons/coming-soon.webp:1]: Don't duplicate.

File: src/pages/products/collapsible-details.js
Comment (leecalcote) [inline src/pages/products/collapsible-details.js:1]: Follow the established paradigm like other pages. Use sections and components.

File: src/assets/images/products/circle_light.svg
Comment (leecalcote) [inline src/assets/images/products/circle_light.svg:1]: Relocate this image.

File: src/assets/images/products/MesheryRange.svg
Comment (leecalcote) [reply src/assets/images/products/MesheryRange.svg:1]: This SVG contains a number of PNGs and is not usable. Ensure only vector contents is used.

File: src/assets/images/products/Rectangle.svg
Comment (leecalcote) [inline src/assets/images/products/Rectangle.svg:1]: This SVG contains a number of PNGs and is not usable. Ensure only vector contents is used.

Don't duplicate the logo.

File: src/assets/images/products/meshery-logo-light.svg
Comment (leecalcote) [inline src/assets/images/products/meshery-logo-light.svg:1]: This SVG contains a number of PNGs and is not usable. Ensure only vector contents is used.

Don't duplicate.

File: src/pages/products/pricing.style.js [hunk 1]
```diff
@@ -0,0 +1,569 @@
+import styled from "styled-components";
```
Comment (leecalcote) [inline src/pages/products/pricing.style.js:1]: A complete copy/paste? We're increasing our sustaining costs.

File: src/sections/General/Navigation/index.js
Comment (leecalcote) [inline src/sections/General/Navigation/index.js:1]: No change, so don't include this file.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Most of the buttons included here don't work, @Modern-artist. 

This PR is not ready for review.

File: src/pages/products/icons/collaboration.svg
Comment (leecalcote) [inline src/pages/products/icons/collaboration.svg:1]: Where is this used?

**Comment by leecalcote** [comment]
The length of the CSS file needs cut down (by ~1/2)

**Comment by l5io** [comment]
🚀 Preview for commit 449a447c555edb9f62ff1de5bc669a0911cf0738 at: https://654ef96c7dbdf35abc87e3ef--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
Hey @Modern-artist  Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by leecalcote** [comment]
@Modern-artist are we pressing forward here or closing out and starting over?

**Comment by Modern-artist** [comment]
> @Modern-artist are we pressing forward here or closing out and starting over?

I would prefer to start again as I haven't written any code for the first two sections.

**Pull request closed without merging**